src/server/database/Database/MySQLConnection.cpp

```diff
@@ -545,8 +545,8 @@ bool MySQLConnection::_HandleMySQLErrno(uint32 errNo, uint8 attempts /*= 5*/)
                 mysql_close(m_Mysql);
                 m_Mysql = nullptr;
             }
+            [[fallthrough]];
```

Comment:
Does this work? GCC complained if the comment wasnt placed right before next case label

Reply:
Got a clean build on GCC 10 and clang 10 (with the changes from #25056 applied). So at least works for me (TM). CI also doesn't seem to mind.